deployments/overmind-kube-source/templates/configmap.yaml

```
@@ -0,0 +1,11 @@
+---
+# ConfigMap definition
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "overmind-kube-source.fullname" . }}-config
+data:
+  LOG: {{ .Values.source.log }}
+  MAX_PARALLEL: {{ .Values.source.maxParallel | quote }}
+  NATS_SERVERS: {{ join "," .Values.source.natsServers }}
```

This is called NATS_URL(S) in other services. Even if those are not public, I'd prefer to be consistent.
```suggestion
  NATS_URLS: {{ join "," .Values.source.natsServers }}
```

---

yikes